cli/tests/pcluster/config/test_param_to_cfn.py

```diff
@@ -30,6 +30,18 @@
         (SCALING, "scaledown_idletime", 10, "10"),
         (SCALING, "scaledown_idletime", 10, "10"),
         (SCALING, "scaledown_idletime", 3, "3"),
+        (
+            CLUSTER,
+            "extra_json",
+            {"cluster": {"cfn_scheduler_slots": "cores"}},
+            '{"cluster": {"cfn_scheduler_slots": "cores"}}',
```

Comment:
here we should have "cfncluster" since it is the result of the `get_cfn_value`. See my previous comment.